### DISCUSSION TITLE
Add support for Rivian

### DISCUSSION THREAD

**Discussion by scottrhoyt** [marcone/teslausb#882] [Ideas]
Rivian has a similar drive cam feature requiring a USB drive. It would be great to use TeslaUSB for these vehicles as well.

**Comment by marcone** [comment]
Somebody started a fork [over here](https://github.com/ebfio/teslausb) (I originally came across this [via Reddit](https://www.reddit.com/r/Rivian/comments/13c7znh/teslausb_but_for_rivian/)) but it looks like they haven't updated it much after that.
